src/components/ImportForm/SampleSection/SampleInfoAlert.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+import { useLocalStorage } from '../../../hooks';
+
+const SAMPLES_FLOW_INSTRUCTION_KEY = 'samples-flow-instruction';
+
+type SampleInfoAlertProps = {
+  localStorageKey: string;
+};
+
+const SamplesInfoAlert: React.FC<SampleInfoAlertProps> = ({ localStorageKey, children }) => {
+  const [storageKeys, setStorageKeys] = useLocalStorage<{ [key: string]: boolean }>(
+    SAMPLES_FLOW_INSTRUCTION_KEY,
+  );
+
+  const keys = storageKeys && typeof storageKeys === 'object' ? storageKeys : {};
+  const isDismissed = keys[localStorageKey];
```

Comment:
Can we store a boolean instead of an object:
```suggestion
const [isDismissed, setIsDismissed] = useLocalStorage<Boolean>(localStorageKey);
```

Reply:
Done.